internal/app/management/module/module.pb.go

```diff
@@ -0,0 +1,651 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
I think it might make sense for these generated grpc files go in the /pkg folder so others can interit and use them, fitting with the standard go structure. But I don't know enough about grpc to know if thats possible or makes sense.